go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/dghubble/go-twitter v0.0.0-20190719072343-39e5462e111f
 	github.com/dghubble/oauth1 v0.6.0
 	github.com/dghubble/sling v1.3.0
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
I'd prefer to not pull in JWT or handle custom JWT formats of different login providers. If they have a simple enough API that gets the User object sure, but if not I'd rather not.

Reply:
See above. It's OIDC so I think there's a broader question as to whether you want gologin to handle validating and parsing userinfo from OIDC tokens for providers who support them, or whether this should be punted to the developer.